### ISSUE TITLE
(Expert) Infinite nether portal tele loop after trying to create twisted saplings

### ISSUE THREAD

**Issue by Zwerkfish** [EnigmaticaModpacks/Enigmatica6#4451]
### Modpack Version

1.0.0

### Describe your issue.

After throwing a stack of spruce saplings through the nether portal to create twisted saplings, i get into a rapid and infinite tele loop between the overworld and nether when going through the nether portal. After a few seconds i start taking damage on each tele, eventually dying through suffocation. There is no grave spawned on death.

Breaking the portal or restarting the client or server does not fix the issue. The only way i found to fix it is by moving the entire portal one block over. 

I was able to reproduce this bug twice, with the portal being in a different location each time.

### Latest Log

https://gist.github.com/Zwerkfish/e70145cbf4b1dc0fadd8dd52830389b7

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by Zwerkfish** [comment]
Update; The looping now also seems to persist after replacing the portal so i was unsure it is related to the saplings after all. I then tried fiddling with my gear and trinkets a bit and figured out the problem is likely cause by the portal charm from Dark Utilities. I tried removing it earlier, which didn't fix the issue, but apparently the effect persists while the item is in your inventory. Weirdly enough the behavior only affects the nether portal, and not others such as the Undergarden and Atum portal.

**Comment by NillerMedDild** [comment]
I can't reproduce this issue in singleplayer, gonna try on a server when I have time :)

**Comment by User-12691296** [comment]
Can confirm. With very low ping, going through a nether portal on a server causes you to teleport to the nether, then back to the overworld, and back, in an infinite loop. The dimension creation lag with low ping causes you to never be able to escape the teleportation loop, as you get teleported before your movements are processed. The suffocation comes from a mod that glitches you into the wall of a nether portal after going through, this used to be a bug in 1.12. Due to the low ping, however, you cannot get pushed out of that block, and instead just stay in it, suffocating to death, unable to do anything. The only fix is rejoining the server.

**Comment by User-12691296** [comment]
The dark utilities charm has nothing to do with it, it just gives you no reaction time to walk back out of the nether portal before getting stuck in the loop

**Comment by theboo** [comment]
yeah, ive had this happen in vanilla with poor connection to server

**Comment by TinyWhenWet** [comment]
This very problem has occurred playing the client on the very same machine the server is hosted on. I do not believe this is caused by low ping.

**Comment by TinyWhenWet** [comment]
> Can confirm. With very low ping, going through a nether portal on a server causes you to teleport to the nether, then back to the overworld, and back, in an infinite loop. The dimension creation lag with low ping causes you to never be able to escape the teleportation loop, as you get teleported before your movements are processed. The suffocation comes from a mod that glitches you into the wall of a nether portal after going through, this used to be a bug in 1.12. Due to the low ping, however, you cannot get pushed out of that block, and instead just stay in it, suffocating to death, unable to do anything. The only fix is rejoining the server.

This very problem has occurred playing the client on the very same machine the server is hosted on. I do not believe this is caused by low ping.
